doc/sphinx/coqdoc.css

```diff
@@ -0,0 +1,338 @@
+             font-size : 115% }
+
+h4.section {
+/*
+                   background-color: rgb(80%,80%,80%);
+                   max-width: 20em;
+                   padding-left: 5px;
+                   padding-top: 5px;
+                   padding-bottom: 5px;
+*/
+                   background-color: white;
+                   padding-left: 0px;
+                   padding-top: 0px;
+                   padding-bottom: 0px;
+                   font-size : 100%;
+                   font-weight : bold;
+                   text-decoration : underline;
+ }
+
+#main .doc { margin: 0px;
+             font-family: sans-serif;
+             font-size: 100%;
+             line-height: 125%;
+             max-width: 40em;
+             color: black;
+             padding: 10px;
+             background-color: #90bdff }
+
+.inlinecode {
+    display: inline;
+/*    font-size: 125%; */
+    color: #666666;
+    font-family: monospace }
+
+.doc .inlinecode {
+    display: inline;
+    font-size: 120%;
+    color: rgb(30%,30%,70%);
+    font-family: monospace }
+
+.doc .inlinecode .id {
+    color: rgb(30%,30%,70%);
+}
+
+.inlinecodenm {
+    display: inline;
+    color: #444444;
+}
+
+.doc .code {
+    display: inline;
+    font-size: 120%;
+    color: rgb(30%,30%,70%);
+    font-family: monospace }
+
+.comment {
+    display: inline;
+    font-family: monospace;
+    color: rgb(50%,50%,80%);
+}
+
+.code {
+    display: block;
+/*  padding-left: 15px; */
+    font-size: 110%;
+    font-family: monospace;
+ }
+
+table.infrule {
+  border: 0px;
+  margin-left: 50px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+td.infrule {
+    font-family: monospace;
+    text-align: center;
+/*    color: rgb(35%,35%,70%);  */
+    padding: 0px;
+    line-height: 100%;
+}
+
+tr.infrulemiddle hr {
+    margin: 1px 0 1px 0;
+}
+
+.infrulenamecol {
+    color: rgb(60%,60%,60%);
+    font-size: 80%;
+    padding-left: 1em;
+    padding-bottom: 0.1em
+}
+
+/* Pied de page */
+
+#footer { font-size: 65%;
+          font-family: sans-serif; }
+
+/* Identifiers: <span class="id" title="...">) */
+
+.id { display: inline; }
+
+.id[title="constructor"] {
+    color: rgb(60%,0%,0%);
+}
+
+.id[title="var"] {
+    color: rgb(40%,0%,40%);
+}
+
+.id[title="variable"] {
+    color: rgb(40%,0%,40%);
+}
+
+.id[title="definition"] {
+    color: rgb(0%,40%,0%);
+}
+
+.id[title="abbreviation"] {
+    color: rgb(0%,40%,0%);
+}
+
+.id[title="lemma"] {
+    color: rgb(0%,40%,0%);
+}
+
+.id[title="instance"] {
+    color: rgb(0%,40%,0%);
+}
+
+.id[title="projection"] {
+    color: rgb(0%,40%,0%);
+}
+
+.id[title="method"] {
+    color: rgb(0%,40%,0%);
+}
+
+.id[title="inductive"] {
+    color: rgb(0%,0%,80%);
+}
+
+.id[title="record"] {
+    color: rgb(0%,0%,80%);
+}
+
+.id[title="class"] {
+    color: rgb(0%,0%,80%);
+}
+
+.id[title="keyword"] {
+ color : #cf1d1d;
+/*     color: black; */
+}
+
+/* Deprecated rules using the 'type' attribute of <span> (not xhtml valid) */
```

Comment:
Is this a problem?

Reply:
This file was copied from Coqdoc, I think